components/teams-service/storage/conformance/conformance_test.go

```diff
@@ -97,7 +97,7 @@ func TestStorage(t *testing.T) {
 
 		authzV2AuthorizationClient := authz_v2.NewAuthorizationClient(authzConn)
```

Comment:
```suggestion
		authzAuthorizationClient := authz_v2.NewAuthorizationClient(authzConn)
```
Also just a few lines above, s/mockV2Authz/mockAuthz/